### PR TITLE
[BEAM-4684] Disable support of @RequiresStableInput on Dataflow runner for now

### PR DESCRIPTION
**Pull request by robinyqiu**
Disable support of @RequiresStableInput on Dataflow runner for now, until we have a more intelligent implementation, as discussed in PR #7991 .

cc: @kennknowles @reuvenlax @rangadi @tweise @mxm 

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python3_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) <br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by robinyqiu** [comment]
Run Java PostCommit

File: runners/google-cloud-dataflow-java/build.gradle [hunk 1]
```diff
@@ -411,22 +411,24 @@ task examplesJavaFnApiWorkerIntegrationTest(type: Test) {
 }
 
 task coreSDKJavaLegacyWorkerIntegrationTest(type: Test) {
-  group = "Verification"
-  dependsOn ":beam-runners-google-cloud-dataflow-java-legacy-worker:shadowJar"
+    group = "Verification"
```
Comment (mxm) [inline runners/google-cloud-dataflow-java/build.gradle]: I think the indention should be 2 here (though not all the groovy code adheres to this).

Comment (robinyqiu) [reply runners/google-cloud-dataflow-java/build.gradle]: Thanks. Fixed.

**Comment by robinyqiu** [comment]
Yes. I was following the discussion on PR #7991. I think we need a way to convey the information that the input to a PTransform is stable.

**Comment by mxm** [comment]
Thanks! Merging this to unblock #7991.

**Pull request merged**